src/Analyzer.Cli.FunctionalTests/Analyzer.Cli.FunctionalTests.csproj

```diff
@@ -33,4 +33,9 @@
     </None>
   </ItemGroup>
 
-</Project>
+  <Target Name="CopyTestFolder" AfterTargets="Build" Condition="!Exists('$(OutputPath)\ADirectoryToAnalyze')">
+    <Exec Command="cp -r ../Analyzer.Cli.FunctionalTests/ADirectoryToAnalyze $(OutputPath)/ADirectoryToAnalyze/" Condition="'$(OS)' == 'Unix'" />
+    <Exec Command="Xcopy /E /I ..\Analyzer.Cli.FunctionalTests\ADirectoryToAnalyze $(OutputPath)\ADirectoryToAnalyze\" Condition="'$(OS)' != 'Unix'" />
+  </Target>
```

Comment:
You shouldn't need this.  Just add additional `<None Update ... >`s above.  You can even do `ADirectoryToAnalyze\*` to output all of them in the directory.